renovate.json5

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+
+    // https://docs.renovatebot.com/presets-npm/#npmunpublishsafe
+    "npm:unpublishSafe"
+  ],
+
+  "commitMessageAction": "update",
+
+  // Tell dependency dashboard to only require PR creation approval for major versions
+  "major": {
+    "dependencyDashboardApproval": true
+  },
+
+  "prConcurrentLimit": 5,
+
+  // Move dependency files forward along with lock files
+  "rangeStrategy": "bump",
+
+  // Allow PR creation between 07:00 and 09:00 UTC on Mondays
+  "schedule": ["* 7-9 * * 1"],
+
+  "packageRules": [
+    {
+      "matchManagers": ["docker-compose"],
+      // "matchUpdateTypes": ["minor", "patch"],
+      // "groupName": "docker-compose dependencies (minor, patch)",
+      "enabled": false // TODO: enable when ready
+    },
+    {
+      "matchManagers": ["dockerfile"],
+      // "matchUpdateTypes": ["minor", "patch"],
+      // "groupName": "dockerfile dependencies (minor, patch)",
+      "enabled": false // TODO: enable when ready
+    },
+    {
+      "matchManagers": ["github-actions"],
```

Comment:
We want these to be delayed by at least a week, and set to pin to hash, is that configurable?

Reply:
Specified a minimum release age in the config. I added (for github actions at least) the `pinDigests` config option, which _should_ handle this, but I'm a little skeptical because I haven't personally used that one before. Worst case, I can go through and replace them all with pinned versions, and from there renovate should carry that policy forward (like you can see in the PR description for aquasecurity/trivy-action)

Reply:
I did some more testing on this on a test repo and it does NOT automatically convert tagged versions to pinned digests. Right now, almost all of the workflows aren't pinning actions versions, but as a follow on PR to this, I could manually convert all of those over to pinned digests and let renovate take over from there.

Reply:
Did more digging. Apparently even if you've got a tag specified, say `uses: renovatebot/github-action@v41.0.17`, if you simply change it to `uses: renovatebot/github-action@v41.0.17 #v41.0.17`, then renovate *will* automatically convert it to a pinned hash:

```
uses: renovatebot/github-action@v41.0.17 # v41.0.17
```

becomes

```
uses: renovatebot/github-action@c21017a4a2fc9f42953bcc907e375a5a544557ec # v41.0.18
```